microservices/blog/src/entities/article.ts

```diff
@@ -54,6 +54,16 @@ class Article {
   @IsNullable()
   publishDate: string | null;
 
+  @JSONSchema({
+    description:
+      'Set of key-value pairs that can be attach to an article.' +
+      'This can be useful for storing extra information about the article in a structured format.',
+  })
+  @Column({ type: 'json', default: {} })
+  @IsUndefinable()
+  @IsObject()
+  extra: Record<string, any>;
```

Comment:
```suggestion
  params: IParams;
```

Reply:
information can differ in different applications